src/averageAges.js

```diff
@@ -55,7 +73,16 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findChild = people.filter(({ mother, sex }) => (onlyWithSon)
```

Comment:
To read name `findChild` I understand that it is a function that after call could find a child. But in this variable stored `Children`.